### PR TITLE
Update to SDK 15.6.5, update UI

### PR DESCRIPTION
**Pull request by duclong201**
- Update to SDK 15.6.5
- Update UI, App icon
- [Jira ticket](https://bluedotinnovation.atlassian.net/jira/software/c/projects/BD/boards/42?modal=detail&selectedIssue=BD-4473)

### CODE REVIEW

**Comment by duclong201** [comment]
This version is using the local package, so after it passes QA and the new SDK version is released. I will update it with NuGet package

**Comment by datkinnguyen** [comment]
> This version is using the local package, so after it passes QA and the new SDK version is released. I will update it with NuGet package

Great. Can you create a ticket for that please?

File: BDPointiOSXamarinDemo/ViewController.cs [hunk 1]
```diff
@@ -83,6 +86,14 @@ private void InitializeSDK()
 
         }
 
+        private void updateVersionLabels()
+        {
+            var appVersion = NSBundle.MainBundle.InfoDictionary[new NSString("CFBundleShortVersionString")].ToString();
+            var buildNumber = NSBundle.MainBundle.InfoDictionary[new NSString("CFBundleVersion")].ToString();
+            appVersionLabel.Text = "App: " + appVersion + "(" + buildNumber + ")";
+            sdkVersionLabel.Text = "SDK: " + locationManager.SdkVersion();
+        }
```
Comment (datkinnguyen) [inline BDPointiOSXamarinDemo/ViewController.cs:95]: could we have a ReleaseNotes file and add this new change to that?

**Review by datkinnguyen — CHANGES_REQUESTED** [review]
could we have a ReleaseNotes file and add this new change to that?

File: BDPointiOSXamarinDemo/ViewController.cs (hunk 1 above)
Comment (duclong201) [reply BDPointiOSXamarinDemo/ViewController.cs:95]: Sure

**Comment by duclong201** [comment]
@datkinnguyen , I have just added ReleaseNotes, it's ready for review now

File: ReleaseNotes.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Release Notes
+
+- Add version labels for debugging
```
Comment (datkinnguyen) [inline ReleaseNotes.md:4]: Would be nice to have the package/SDK version in Release notes too

**Review by datkinnguyen — APPROVED** [review]

**Pull request merged**